content/800-guides/330-management-api-basic.mdx

```diff
@@ -0,0 +1,196 @@
+---
+title: 'Get started with the Prisma Management API'
+metaTitle: "Get started with the Prisma Management API"
+description: "Learn how to get started with the Prisma Management API"
+sidebar_label: 'Management API starter'
+image: '/img/guides/neon-connection-pooling.png'
+completion_time: '15 min'
+community_section: true
+---
+
+## Overview
+
+This guide walks you through setting up a basic TypeScript project that uses the [Prisma Management API](/postgres/database/api-reference/management-api) to create a new [Prisma Console project](/platform/about#project) with a [Prisma Postgres](/postgres/introduction/overview) database, and print out all connection details. 
+
+You'll authenticate via a [service token](/postgres/database/api-reference/management-api#bearer-tokens), set up your environment, and run a script to interact with the API.
+
+## Prerequisites
+
+- Node.js and `npm` installed
+- A [Prisma Data Platform](https://cloud.prisma.io/) account
+
+## 1. Create a service token in Prisma Console
+
+First, you need to create a service token to be able to access your workspace in Prisma Console:
+
+1. Open the [Prisma Console](https://cloud.prisma.io/)
+2. Navigate to the **Integrations** page of your workspace
+3. Click **Generate integration token**
+4. Copy and save the generated service token securely, you'll use it in step 2.2.
+
+## 2. Set up your project directory
+
+### 2.1. Create a basic TypeScript project
+
+Open your terminal and run the following commands:
+
+```terminal
+mkdir management-api-demo
+cd management-api-demo
+```
+
+Next, initialize npm and install dependencies required for using TypeScript:
+
+```terminal
+npm init -Y
```

Comment:
```suggestion
npm init -y
```

Reply:
We can change it to small `y` so that users won't have to press enter multiple times to create the `package.json`. In case of capital `Y` you are explicitly asked to name details of package.json. With `y` it adds everything by default.

Reply:
thanks Nurul!!